test/fixtures/test_runner_fixtures.dart

```diff
@@ -1120,6 +1120,21 @@ const skipExceptionMessageJsonOuput = [
     'type': 'testStart',
     'time': 2757
   },
+  {
+    'test': {
```

Comment:
can we move this test after id: 28 so that they are in order? Also we should update the testDone event to have a different timestamp

Reply:
Thank you @felangel  for reviewing.  Done

Reply:
Dear Reviewers @felangel  @tomarra  @erickzanardo  @dewinjm , if you have a bit littre time to reviews my 4 PRs. We need it in our projet. Thank you in advance